test/lightning/usage_tracking/configuration_management_service_test.exs

```diff
@@ -0,0 +1,233 @@
+defmodule Lightning.UsageTracking.ConfigurationManagementServiceTest do
+  use Lightning.DataCase
+
+  alias Lightning.Repo
+  alias Lightning.UsageTracking.ConfigurationManagementService
+  alias Lightning.UsageTracking.DailyReportConfiguration
+
+  describe ".enable/1 - no configuration exists" do
+    setup do
+      {:ok, tracking_enabled_at, _offset} =
+        DateTime.from_iso8601("2024-03-01T18:23:23.000000Z")
+
+      start_reporting_after = Date.from_iso8601!("2024-03-01")
+
+      %{
+        tracking_enabled_at: tracking_enabled_at,
+        start_reporting_after: start_reporting_after
+      }
+    end
+
+    test "creates record", config do
+      %{
+        tracking_enabled_at: tracking_enabled_at,
+        start_reporting_after: start_reporting_after
+      } = config
```

Comment:
@jyeshe I try to do that when I can fit it in on a single line but the formatting that the formatter requires when the config overflows breaks auto-indentation in my editor and I am continually having to correct the indentation.  of all the things I like about Elixir, the formatter is most decidedly not one of them :).

I will correct.

Reply:
hey @rorymckinley line breaks and formatting are allowed it would be only about the style, pattern matching the test context right after the test name.